source/guides/community-for-mattermost.rst

```diff
@@ -8,26 +11,25 @@ Use the Mattermost for Microsoft Teams integration to stay connected with thousa
 .. image:: ../images/mattermost-for-microsoft_365.png
   :alt: Mattermost for Microsoft 365 is available in the Microsoft App Store.
 
-Key features
-------------
-
-- **Direct Access**: Access the Community for Mattermost directly from a tab without switching applications or opening a browser.
-- **Seamless Integration**: Experience the full functionality of Community for Mattermost within a familiar interface.
-- **Real-time Collaboration**: Engage with the Community for Mattermost in real-time discussions on product features, technical questions, and best practices.
-- **Product Support**: Get help from both Mattermost staff and experienced community members.
-- **Contribute to Development**: Participate in discussions that shape the future direction of Mattermost products.
-- **Knowledge Sharing**: Learn implementation strategies and best practices from a diverse community of users.
-- **Stay Updated**: Keep up with the latest Mattermost announcements, updates, and roadmap information.
-
-This app is designed to work with Microsoft 365, Outlook, and Microsoft Teams.
+This app is designed to work with Microsoft 365, Outlook, and Teams. A free Mattermost account is required to use the Mattermost Customer Community.
```

Comment:
I think we're still calling it the `Mattermost Community`.  It's more than customers.